src/pages/blamefile.js

```diff
@@ -1,20 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Loading from '../components/Loading';
 import { fetchBlameFileData } from '../utils/blamefileQueries/blamefileDataFetcher';
+import { validateRepoStats } from '../utils/validation/validate';
+import Tables from '../components/Tables';
 
-const BlameFile = () => {
-    const handleSubmit = (e) => {
+function History(props) {
+    return <li> 
+        {props.startingLine}
+        {props.endingLine}
+        {props.age}
+        {props.url}
+        <ul>
+            {props.oid}
+            {props.name}
+            {props.committedDate}
+            {props.totalCount}
+        </ul>
+    </li>
+}
+
+function BlameFile() {
+    const [error, setErrors] = useState({ ownername: { status: false, message: "" }, reponame: { status: false, message: "" }, ownername: { status: false, message: "" }, filepath: { status: false, messasge: "" } })
+    const [form, setForm] = useState({ ownername: "CSC4350-TR", reponame: "GitHubGo", filepath: "src/App.js" });
+    const [validate, setValidate] = useState({ ownername: false, reponame: false, filename: true })
+    // const [isUser, setUser] = useState("org")
+    const [fetchData, setFetchData] = useState({ userid: "", branches: new Set([]) })
+    
+    const [blamefileData, setfetchBlameFileData] = useState({ startingLine: 0, endingLine: 0, age: 0, name: "", oid:0, date:new Date().toLocaleString(), total:0, url:"", })
```

Comment:
```suggestion
const [blamefileData, setfetchBlameFileData] = useState({ startingLine: 0, endingLine: 0, age: 0, name: "", oid:0, date: "", total:0, url:"", })
```
Since you are accepting the date as a string, you can keep it empty, since this won't be displayed anyways.